dali/base/dasess.cpp

```diff
@@ -1118,6 +1152,16 @@ class CClientSessionManager: public CSessionManagerBase, implements IConnectionM
         queryCoven().sendRecv(mb,RANK_RANDOM,MPTAG_DALI_SESSION_REQUEST,SESSIONREPLYTIMEOUT);
     }
 
+    virtual void refreshWhiteList() override
+    {
+        assertex(!"refreshWhiteList called on client");
```

Comment:
minor: more recently throwUnexpected("refreshWhiteList called on client"); tends to be how this would be coded

Reply:
yeah agree, but would then have wanted to change all the other cases in this file and decided against it at this stage.

Will change this and others in this file now though.